back-end/index.js

```diff
@@ -1,32 +1,46 @@
 import app from "./server.js"
-import mongodb from "mongodb"
+// import mongodb from "mongodb"
+import mongoose from "mongoose"
 import dotenv from "dotenv"
-import TranscriptsDAO from "./dao/transcriptsDAO.js"
-import TextTranscriptsDAO from "./dao/textTranscriptsDAO.js"
-import ProjectsDAO from "./dao/projectsDAO.js";
+// import TranscriptsDAO from "./dao/transcriptsDAO.js"
+// import TextTranscriptsDAO from "./dao/textTranscriptsDAO.js"
+// import ProjectsDAO from "./dao/projectsDAO.js";
 
 dotenv.config()
-const MongoClient = mongodb.MongoClient
+// const MongoClient = mongodb.MongoClient
 
-const port = process.env.PORT || 8000
+const port = process.env.PORT || 5000
```

Comment:
why was the port switched?

Reply:
Note that our PORT env variable was referring to port 8000 anyways, so the port was always being set to 8000, but with this, let's say the env var wasn't present in the env; the port would be set to 5000 instead.

Reply:
Makes sense